awx/main/utils/reload.py

```diff
@@ -4,25 +4,24 @@
 # Python
 import subprocess
 import logging
+import os
 
-# Django
-from django.conf import settings
 
 logger = logging.getLogger('awx.main.utils.reload')
 
 
-def _supervisor_service_command(command, communicate=True):
+def supervisor_service_command(command, service='*', communicate=True):
     '''
     example use pattern of supervisorctl:
     # supervisorctl restart tower-processes:receiver tower-processes:factcacher
     '''
-    group_name = 'tower-processes'
-    if settings.DEBUG:
-        group_name = 'awx-processes'
     args = ['supervisorctl']
-    if settings.DEBUG:
-        args.extend(['-c', '/supervisor.conf'])
-    args.extend([command, '{}:*'.format(group_name)])
+
+    supervisor_config_path = os.getenv('SUPERVISOR_WEB_CONFIG_PATH', None)
```

Comment:
Add to blacklist https://github.com/ansible/awx/blob/0f1a92bd515b2ca5f5fa5451fe49f58c2e7f6cda/awx/main/constants.py#L35

Reply:
good eyes 👍 we should address this @rooftopcellist

Reply:
✅ @rooftopcellist let's track this as a task post-merge